### PR TITLE
Setup exercise templates

### PR DESCRIPTION
**Pull request by alvara**
backend work to get ai working

### CODE REVIEW

File: app/helpers/ai_helper.rb [hunk 1]
```diff
@@ -98,6 +157,7 @@ def ai_create_workout(user_query)
         exercise.station = Station.all.sample
```
Comment (david-mambou) [inline app/helpers/ai_helper.rb:157]: Assigning a random station to each exercise?

File: app/controllers/workouts_controller.rb [hunk 2]
```diff
@@ -59,15 +48,6 @@ def update
     end
   end
 
-  def mark_finished
```
Comment (david-mambou) [inline app/controllers/workouts_controller.rb]: Method removed, but did you remove the corresponding method calls? (cf workouts/show view)
Also if you're sure we don't need this anymore you can remove the route and the policy

**Review by david-mambou — COMMENTED** [review]
Check the two comments please

File: app/helpers/ai_helper.rb (hunk 1 above)
Comment (alvara) [reply app/helpers/ai_helper.rb:157]: think this was done earlier just to satisfy the foreign key for seeding at the time, but michael has an update with more advance seeds , so i imagine this random station will be removed in future when his code is merged.

**Review by david-mambou — APPROVED** [review]

**Pull request merged**